app/templates/jobs/new.hbs

```diff
@@ -106,59 +120,57 @@
           </AuLabel>
           <AuInput id="task-creator" @width="block" @value={{this.creator}} disabled={{true}} {{! template-lint-disable no-duplicate-id }} />
         </div>
-        <AuButton class="au-u-margin-top-large" {{on 'click' this.scheduleImportJob}}>Schedule</AuButton>
+        <AuButton
+          class="au-u-margin-top-large"
+          {{on 'click' this.scheduleImportJob}}
+          @disabled={{this.scheduling}}
+          @loading={{this.scheduling}}
```

Comment:
FYI, the latest Appuniversum v2 release also disables the button when `@loading={{true}}`. It might be worth updating the app to the latest version if you're adding new features to it.